src/Utils/PluginInfo.as

```diff
@@ -70,8 +71,9 @@ class PluginInfo
 		auto jsScreenshots = js["screenshots"];
 		for (uint i = 0; i < jsScreenshots.Length; i++) {
 			auto jsScreenshot = jsScreenshots[i];
-			if (jsScreenshot.GetType() == Json::Type::String) {
-				m_screenshots.InsertLast(jsScreenshot);
+			if (jsScreenshot.GetType() == Json::Type::Object) {
+				m_screenshots.InsertLast(jsScreenshot["uri"]);
+				m_screenshotDescriptions.InsertLast(NullCoalesce(jsScreenshot["description"]));
```

Comment:
We should check for both string & object for now, so that in the website transition period this will still work as expected.

Reply:
pushed a fix for the old API format!